azure-pipelines.yml

```
@@ -92,7 +92,18 @@ jobs:
     inputs:
       command: 'build'
       projects: '$(Build.SourcesDirectory)/nanoFirmwareFlasher/nanoFirmwareFlasher.Tool/nanoFirmwareFlasher.Tool.csproj'
-      arguments: '-o $(Build.SourcesDirectory)/dist/utils/nanoFirmwareFlasher /p:DefineConstants=VS_CODE_EXTENSION_BUILD /p:RestoreLockedMode=false /p:VSCodeExtensionBuild=True'
+      arguments: '-o $(Build.SourcesDirectory)/dist/utils/nanoFirmwareFlasher /p:RestoreLockedMode=false'
```

These arguments really need to be there. We don't want the custom build to behave like the regular nanoff.
```suggestion
      arguments: '-o $(Build.SourcesDirectory)/dist/utils/nanoFirmwareFlasher /p:DefineConstants=VS_CODE_EXTENSION_BUILD /p:RestoreLockedMode=false /p:VSCodeExtensionBuild=True'
```